src/components/SelectionPane.jsx

```diff
@@ -1,37 +1,38 @@
 import React from "react";
 import styled from "styled-components";
+import PropTypes from "prop-types";
 
 import CollapsableHeader from "./CollapsableHeader";
 import SelectionCard from "./SelectionCard";
 
-import { getFaction } from "../util/jsonUtils";
-
-import { FACTIONS } from "../util/constants";
-
 const Body = styled.div`
   border-style: solid;
   border-width: 1px;
   flex-grow: 1;
   border-right: 0;
 `;
 
-const Category = ({ title, units }) => (
+const Category = ({ title, units, addFunc }) => (
```

Comment:
Consider if there's a better name for `addFunc` like `addUnitTo<Roster/Team/Squadron>` or something